crates/matrix-sdk-crypto/src/gossiping/machine.rs

```diff
@@ -953,36 +959,53 @@ impl GossipMachine {
                     sender_key = sender_key.to_base64(),
                     claimed_sender_key = content.claimed_sender_key.to_base64(),
                     room_id = content.room_id.as_str(),
+                    %algorithm,
                     "Couldn't create a group session from a received room key"
                 );
                 Err(e.into())
             }
         }
     }
 
+    async fn receive_supported_keys(
```

Comment:
I understand this is extracted from `receive_forwarded_room_key`, but wouldn't it be great to add a documentation even if the method isn't public, thoughts?

Reply:
Eh, I expect this to move back into `receive_forwarded_room_key` by the time we can remove the feature flag. But sure, I'll add a sentence or two.